projects/sandbox/src/app/sandbox-navigation/sandbox-navigation.component.ts

```diff
@@ -572,7 +559,7 @@ export class SandboxNavigatonComponent extends DataPollingComponent implements O
           this.problemPatternsDataset = undefined;
           confStep.error = err;
           confStep.lastLoadedIdDataset = undefined;
-          this.resetBusy();
+          confStep.isBusy = false;
```

Comment:
just out of curiosity `isProcessing` better word than `isBusy`?

Reply:
Well, it could be loading or polling or (coming soon) processing the result... in this case `isLoading` may be better, but `isBusy` is nice and generic!